src/SFA.DAS.IdentifyDataLocks.Domain/CollectionPeriodMappingExtensions.cs

```diff
@@ -60,9 +65,16 @@ public static class CollectionPeriodMappingExtensions
                 Cost = earning.CalculateCost(),
                 PriceStart = earning.PriceEpisodes.FirstOrDefault()?.StartDate,
                 StoppedOn = earning.PriceEpisodes.FirstOrDefault()?.ActualEndDate,
-                IlrSubmissionDate = earning.IlrSubmissionDateTime
+                IlrSubmissionDate = earning.IlrSubmissionDateTime,
+                Tnp1 = GetTnpValue(earning.PriceEpisodes.FirstOrDefault()?.TotalNegotiatedPrice1),
+                Tnp2 = GetTnpValue(earning.PriceEpisodes.FirstOrDefault()?.TotalNegotiatedPrice2),
+                Tnp3 = GetTnpValue(earning.PriceEpisodes.FirstOrDefault()?.TotalNegotiatedPrice3),
+                Tnp4 = GetTnpValue(earning.PriceEpisodes.FirstOrDefault()?.TotalNegotiatedPrice4)
             };
 
+        private static string GetTnpValue(decimal? tnp) => 
+            tnp.GetValueOrDefault() > 0 ? tnp.Value.ToString("c") : "";
```

Comment:
🤔 if there is a zero value specified in the ILR - instead of not having any value - do you think we should show that zero in the report, rather than a blank?

I think we should probably show the values that were actually submitted, if present.